umh.docs.umh.app/content/en/docs/features/ifm-retrofitting/index.md

```diff
@@ -1,24 +1,51 @@
 +++
-title = "Retrofitting with ifm IO-link gateways"
-menuTitle = "Retrofitting with ifm IO-link gateways"
+title = "Retrofitting with ifm IO-link gateways and sensorconnect"
+menuTitle = "Retrofitting with ifm IO-link gateways and sensorconnect"
 description = ""
 weight = 4
-draft = true
+draft = false
 +++
 
-Introduction sentence
+Retrofitting older machines with sensors is sometimes the only-way to capture process-relevant information.
+In this article, we will focus on retrofitting with ifm IO-link gateways and Sensorconnect, a microservice of the 
+united manufacturing hub, that finds and reads out ifm IO-link masters in the network and pushes sensor data to 
+MQTT/Kafka for further processing.
+
+![](/images/features/ifm-retrofitting/ifm_sensors.jpg?width=40%)
 
 ## When should I use it?
 
+Retrofitting with ifm IO-link gateways and using Sensorconnect is ideal when dealing with older machines that are not
+equipped with any connectable hardware to read relevant information out of the machine itself. By placing sensors on 
+the machine and connecting them with ifm IO-link gateways, required information can be gathered for valuable
+insights. Sensorconnect helps to easily connect to all sensors to correctly and properly capture the large 
+amount of sensor data provided.
 
 ## What can I do with it?
 
+With ifm IO-link gateways and Sensorconnect, you can collect data from sensors and make it accessible for further use. 
+Sensorconnect offers: 
+- automatic detection of ifm IO-link master in the network
+- identification of IO-Link or other digital sensors connected to the gateways
+- constant polling of data from the detected sensors with a high frequency
+- interpreting the received data based on a sensor database containing thousands of entries
+- sending data in JSON format to MQTT and Kafka for further data processing
+
 
 ## How can I use it?
 
+To use ifm IO-link gateways and [Sensorconnect](/docs/architecture/microservices/core/sensorconnect/), you need to first
```

Comment:
Do a step-by-step tutorial here

1. Connect the IO-link gateways to the same network
2. Retrofit the machine with sensors and connect them to the IO-link gateway
3. Configure the sensorconnect ip range to either match the IP address using subnet notation `/32`, or if you have multiple masters, configure it to scan an entire range `/24/
4. Data should appear in your Unified Namespace